planner/core/physical_plans.go

```diff
@@ -2058,6 +2098,13 @@ type PhysicalSelection struct {
 	// The flag is only used by cost model for compatibility and will be removed later.
 	// Please see https://github.com/pingcap/tidb/issues/36243 for more details.
 	fromDataSource bool
+
+	// todo Since the feature of adding filter operators has not yet been implemented,
+	// the following code for this function will not be used for now.
+	// The flag indicates whether this Selection is used for RuntimeFilter
+	// True: Used for RuntimeFilter
+	// False: Only for normal conditions
+	// hasRFConditions bool
```

Comment:
Just curious, what's this for?

Reply:
There are two filter conditions on selection, one is runtime filter and the other is common filter. 
This field is mainly used to identify that the current selection contains predicates of type runtime filter. 
After marking, selection will wait for the RF status to become ready before executing.

Reply:
A bit confused.
So do we need this field to make the runtime filter work correctly?
I thought the newly added fields in `PhysicalTableScan` is enough.

Reply:
This is a field for tiflash. The compile stage of tiflash, it use this field to recognize special selection